src/Core/Services/ApiService.cs

```diff
@@ -177,7 +177,7 @@ public Task PostAccountVerifyPasswordAsync(PasswordVerificationRequest request)
             return SendAsync<PasswordVerificationRequest, object>(HttpMethod.Post, "/accounts/verify-password", request,
                 true, false);
         }
-
+        
```

Comment:
Looks like perhaps your IDE is added these extra whitespace perhaps?

Reply:
I don't believe so. Every other region has the same spacing except this one, so just added it to be uniform with the others.